frontend.sh

```diff
@@ -12,83 +12,101 @@ set -e
 # Initialize project dependency directories.
 init() {
   # Set cli_flag variable.
-  source cli-flag.sh 'Front end' $1
+  source cli-flag.sh "Front-end" $1
 
-  if [ -f "package-lock.json" ]; then
-    DEP_CHECKSUM=$(cat package*.json cfgov/unprocessed/apps/**/package*.json | shasum -a 256)
-  else
-    DEP_CHECKSUM=$(cat package.json cfgov/unprocessed/apps/**/package.json | shasum -a 256)
-  fi
-
-  if [[ "$(node -v)" != 'v8.'* ]]; then
-    printf "\033[1;31mPlease install Node 8.x: 'nvm install 8'\033[0m\n"
+  if [[ "$(node -v)" != "v8."* ]]; then
+    echo "Please install Node 8.x: 'nvm install 8'"
     exit 1;
   fi
-
-  NODE_DIR=node_modules
-  echo "npm components directory: $NODE_DIR"
 }
 
 # Clean project dependencies.
 clean() {
   # If the node directory already exists,
   # clear it so we know we're working with a clean
   # slate of the dependencies listed in package.json.
-  if [ -d $NODE_DIR ]; then
-    echo 'Removing project dependency directories…'
-    rm -rf $NODE_DIR
-    echo 'Project dependencies have been removed.'
+  if [ -d node_modules ]; then
+    echo "Removing project dependency directories…"
+    rm -rf node_modules
+    echo "Project dependencies have been removed."
   fi
 }
 
 # Install project dependencies.
 install() {
-  echo 'Installing front-end dependencies…'
+  echo "Installing front-end dependencies…"
 
   if [ "$cli_flag" = "development" ]; then
 
     npm install -d --loglevel warn
 
     # Protractor = JavaScript acceptance testing framework.
-    echo 'Installing Protractor dependencies locally…'
+    echo "Installing Protractor dependencies locally…"
     # We skip Gecko here (--gecko false) because webdriver pulls its release
     # directly from a GitHub.com URL which enforces rate-limiting. This can
     # cause installation failures when running automated testing. Currently
     # we don't rely on Gecko for testing.
-    ./$NODE_DIR/protractor/bin/webdriver-manager update --gecko false --standalone false
+    ./node_modules/protractor/bin/webdriver-manager update --gecko false --standalone false
 
   else
     npm install --production --loglevel warn --no-optional
   fi
 }
 
+# Calculate checksum value.
+calc_checksum() {
+  if [ -f "package-lock.json" ]; then
+    DEP_CHECKSUM=$(cat package*.json cfgov/unprocessed/apps/**/package*.json | shasum -a 256)
+  else
+    DEP_CHECKSUM=$(cat package.json cfgov/unprocessed/apps/**/package.json | shasum -a 256)
```

Comment:
We can probably just use 

`DEP_CHECKSUM=$(cat package*.json cfgov/unprocessed/apps/**/package*.json | shasum -a 256)`

and get rid of the if / else statment.